index.js

```diff
@@ -25,6 +25,14 @@ const Bridge = class Bridge extends EventEmitter {
       return Object.assign({}, message.payload_fields, metadata);
     }
 
+    const appId = checkSetting(process.env.TTN_APP_ID);
+    const accessKey = checkSetting(process.env.TTN_APP_ACCESS_KEY);
+    const region = checkSetting(process.env.TTN_REGION);
+
+    const hubName = checkSetting(process.env.TTI_AZURE_HUBNAME);
+    const keyName = checkSetting(process.env.TTI_AZURE_KEYNAME);
+    const key = checkSetting(process.env.TTI_AZURE_KEY);
```

Comment:
Pass them still as parameters, see previous comment, and just check here with `if (!appId || !accessKey || ....)`